src/dali_backend.cc

```diff
@@ -484,10 +498,6 @@ TRITONSERVER_Error* TRITONBACKEND_ModelInstanceExecute(TRITONBACKEND_ModelInstan
             responses[i], TRITONSERVER_ResponseCompleteFlag::TRITONSERVER_RESPONSE_COMPLETE_FINAL,
             error),
         make_string("Failed sending response, idx ", i));
-    LOG_IF_ERROR(
```

Comment:
Why removed?

Reply:
Because TritonRequest class handles releasing the request in RAII fashion